source/_integrations/sms.markdown

```diff
@@ -0,0 +1,74 @@
+---
+title: "SMS"
+description: "SMS notification via GSM modem."
+icon: gammu.png
+ha_category:
+  - Notifications
+ha_release: 0.105
+ha_iot_class: Local Polling
+ha_config_flow: false
+ha_codeowners:
+  - '@ocalvo'
+---
+
+The `sms` integration allows having a local execution SMS notification via [Gammu](https://wammu.eu/gammu/). This is ideal when the internet is offline or when the power goes out.
+
+This integration provides the following platforms:
+- Notify
+
+## Configuration
+
+To enable those notifications in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sms:
+  device: /dev/ttyUSB2
+
+notify:
+  - platform: sms
+    name: sms_person1
+    phone_number: PHONE_NUMBER
+  - platform: sms
+    name: sms_person2
+    phone_number: PHONE_NUMBER
+```
+{% configuration %}
+device:
+  description: The gsm modem device.
+  required: true
+  type: string
+{% endconfiguration %}
+
+To use notifications, please see the [getting started with automation page](/getting-started/automation/).
+
+It requires HassOS version [3.6](https://github.com/home-assistant/hassos/releases/tag/3.6) or higher.
```

Comment:
Still have the same comment as before (which you've marked resolved without comment). Can't it be used by non-HassOS users?

Reply:
Yes, this can be used it by non-HassOS users

Reply:
Added more info for non-Hass.io users